app/src/test/java/ru/vssemikoz/newsfeed/newsfeed/NewsFeedPresenterTest.java

```diff
@@ -26,11 +29,13 @@
 import static org.junit.Assert.assertEquals;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.timeout;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 @RunWith(MockitoJUnitRunner.class)
 public class NewsFeedPresenterTest {
+    private static final int TIME_OUT_SECONDS = 2000;
```

Comment:
A timeout placed in the unit tests is a code smell. You should re-write your tests to let them work without it. It should only be used if you're dealing with concurrency, but your case doesn't have any issues with it.